worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -453,6 +453,14 @@ private[celeborn] class Worker(
       0
     }
   }
+  // Unreleased partition location count when worker is restarting
+  workerSource.addGauge(WorkerSource.UNRELEASED_PARTITION_LOCATION_COUNT) { () =>
+    if (shutdown.get() && (workerStatusManager.currentWorkerStatus.getState == State.InGraceFul)) {
```

Comment:
The condition `workerStatusManager.currentWorkerStatus.getState == State.InGraceFul `can be removed since there may be other states when shutdown.

Reply:
done